client/src/services/lab7/RepairService.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
```

Comment:
same here, repair is separate from exercise input

Reply:
Done.